corehq/apps/translations/tasks.py

```diff
@@ -166,7 +166,7 @@ def download_project_from_hq(domain, data, email):
     gen = AppTranslationsGenerator(domain, data.get('app_id'), data.get('version'), lang, lang, 'default_')
     parser = TranslationsParser(gen)
     try:
-        translation_file, filename = parser.generate_excel_file()
+        translation_file, __ = parser.generate_excel_file()
```

Comment:
Guessing this a double underscore to avoid conflicting with a ugettext alias, which is clever but also kind of unreadable, might be better to just name it `result_file` and then not use it.

Reply:
A single underscore is [convention](https://stackoverflow.com/a/5477153) for an ignored unpacked variable, but because it's used in Django as shorthand for `gettext`, people tend to use a double underscore instead.

Reply:
Got it. I've seen the underscore convention, but it makes me a little sad that double underscore has a following.

Reply:
lol, oh dear.